freqtrade/rpc/telegram.py

```diff
@@ -553,6 +561,8 @@ def _help(self, update: Update, context: CallbackContext) -> None:
                   "*/stop:* `Stops the trader`\n" \
                   "*/status [table]:* `Lists all open trades`\n" \
                   "         *table :* `will display trades in a table`\n" \
+                  "                  pending buy orders are marked with an asterisk (*)\n" \
+                  "                  pending sell orders are marked with a double asterisk (**)\n" \
```

Comment:
something in the first of these 2 lines is causing /help to fail.

```
2020-02-13 06:48:06,063 - freqtrade.rpc.telegram - WARNING - Telegram NetworkError: Can't parse entities: can't find end of the entity starting at byte offset 947! Trying one more time.
2020-02-13 06:48:06,141 - freqtrade.rpc.telegram - WARNING - TelegramError: Can't parse entities: can't find end of the entity starting at byte offset 947! Giving up on that message.
```

it's not the length of the message (2 times the 2nd line does work).
I've even tried to retrype it to no avail ... 

Also - to align visualization - i think both lines should be quoted in single tics (which seems to fix the above case) - otherwise this is the only section in help that's not shown as code - or in bold.

I've done that modification - hope that's ok for you.

Reply:
Gotcha. Thanks.